Remotion/ObjectBinding/Web.IntegrationTests/BocAutoCompleteReferenceValue/ReadOnlyBocAutoCompleteReferenceValueTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.ControlObjects;
+using Remotion.Web.Development.WebTesting.ControlObjects.Selectors;
+using Remotion.Web.Development.WebTesting.ControlSelection;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocAutoCompleteReferenceValue
+{
+  [TestFixture]
+  public class ReadOnlyBocAutoCompleteReferenceValueTest : IntegrationTest
+  {
+    [Test]
+    public void Validate_OnReadOnlyControl_CreatesValidationErrors ()
+    {
+      var home = Start();
+      var webButtonControlObject = home.GetControl(new LocalIDControlSelectionCommand<WebButtonControlObject>(new WebButtonSelector(), "ValidateButton"));
```

Comment:
And big picture code style: you could add an extension method to get the ValidateButton (`home.ValidateButton()`) to the teest assembly to remove this code duplication on how the test page works.

Reply:
Hm... someone already did that, I just didn't find it.